plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/internal/AppYamlValidatorTest.java

```diff
@@ -67,6 +67,15 @@ public void testContructor_nonAbsoluteBasePath() {
   }
 
   @Test
+  public void testValidate_emptyField() {
+    when(appYamlPath.getValue()).thenReturn("");
+
+    IStatus result = pathValidator.validate();
+    assertEquals(IStatus.ERROR, result.getSeverity());
+    assertEquals("Enter app.yaml path.", result.getMessage());
```

Comment:
Is it possible to enter the path where this message is seen?
If not, it should say something like "Missing app.yaml path."